spread.yaml

```diff
@@ -601,26 +609,30 @@ restore-each: |
     if journalctl -u snapd.service | grep -F "signal: terminated"; then exit 1; fi
 
 suites:
-    tests/main/:
-        summary: Full-system tests for snapd
+    # The essential tests run designed to run inside the autopkgtest
+    # environment on each platform. On autopkgtest cannot run all tests
+    # as this is very slow and we run into timeouts.
+    #
+    # These tests are run on all other plattforms as well first as they
```

Comment:
the "as well" feels confusing/unnecessary here

Reply:
updated